src/azure-cli/azure/cli/command_modules/appservice/azure_devops_build_interactive.py

```diff
@@ -115,7 +115,7 @@ def azure_devops_flow(self):
         self.process_build('AZURE_DEVOPS')
         self.wait_for_build()
         self.process_release()
-        self.logger.warning("Pushing your code to master will now trigger another build.".format(
+        self.logger.warning("Pushing your code to {remote}:master will now trigger another build.".format(
```

Comment:
I don't understand this fix. Could you share the knowledge?

Reply:
F522 '...'.format(...) has unused named argument(s): remote